src/views/Community/View.js

```diff
@@ -0,0 +1,141 @@
+import React, { Component } from 'react'
+// import Dashboard from '../Dashboard'
+import { connect } from 'react-redux'
+import { compose } from 'redux'
+import { Row, Progress } from 'reactstrap';
+import { sendCommunityData } from '../../store/actions/community'
+import Payment from '../../components/Payment'
+import { isAuthUserType, isAuthEmail, removeAuthenticatedState } from '../../services/Auth'
+import { Link } from 'react-router-dom'
+import Page404 from '../../views/Pages/Page404'
```

Comment:
Codacy found an issue: [Useless path segments for "../../views/Pages/Page404", should be "../Pages/Page404"](https://app.codacy.com/gh/BuildForSDG/Team-064-Frontend/pullRequest?prid=5714105)